test/test.js

```diff
@@ -223,6 +223,27 @@ describe('launder', function() {
     it('should return the default if it is malicious', function() {
       assert(launder.url('javascript:alert(\'All your base are belong to us\');', 'http://www.apostrophenow.org') === 'http://www.apostrophenow.org');
     });
+    it('should return an https url', function() {
+      assert(launder.url('sms://www.apostrophenow.org') === 'sms://www.apostrophenow.org');
```

Comment:
Please change this test to match the description. 'https', not 'sms'.

Reply:
Updated it to https